### ISSUE TITLE
Not Include EXCLUDED_ARCHS in Xcode12

### ISSUE THREAD

**Issue by zmjios** [CocoaPods/Xcodeproj#784]
in XCBuildConfiguration not include EXCLUDED_ARCHS,beacse in Xcode12, Apple has been remove VALID_ARCHS,more details:https://developer.apple.com/documentation/xcode-release-notes/xcode-12-beta-release-notes

**Comment by dnkoutso** [comment]
I do not understand a 100% the issue report. Should we be adding `EXCLUDED_ARCHS`?

**Comment by zmjios** [comment]
yeah,https://github.com/CocoaPods/Xcodeproj/blob/15c9005e6c96892db5c96ac6323894f4c73d9dea/spec/project/object/native_target_spec.rb,native_target_spec may be add EXCLUDED_ARCHS
